newIDE/app/public/index.html

```diff
@@ -61,5 +61,24 @@
         </p>
       </div>
     </noscript>
+    <script>
+      (function() {
+        // Apply background color for startup
+        var backgroundColors = {
+          Dark: '#252525',
+          Nord: '#2E3440',
+          'Solarized Dark': '#00212B'
+        };
+        var themeName = JSON.parse(localStorage.getItem("gd-preferences"))['themeName'];
+        if (!(themeName in backgroundColors)) {
+          console.log('Theme does not have a background color');
+          alert('Theme does not have a background color');
+          return;
+        }
+        if (themeName != 'GDevelop default') {
+          document.body.style.backgroundColor = backgroundColors[themeName]
```

Comment:
We do this only at the startup. I wonder if we should also do this inside the app when changing theme, as the background color could be visible some times?

Reply:
(see for example how we do for the titlebar color for PWAs in `ProjectTilebar.js` - we could do the same with the document.body.style.backgroundColor and the Electron background color: https://www.electronjs.org/docs/api/browser-window#winsetbackgroundcolorbackgroundcolor)

Reply:
You've not addressed this comment before asking for a review again :)

Reply:
Sorry about that! I was having a hard time figuring out where to put the code to do that.

Reply:
You've again requested a review while this is not addressed: please rather ask questions rather than requesting review :)

Reply:
(if you can't find how to do it it's no problem, I'll do it or explain more but prefer to ask questions rather than just asking for review again, to avoid losing time for both of us)

Reply:
I think I know how to do it, I just don't know where it should go in the codebase.

Reply:
@4ian I actually think this is not too necessary because the main `div` takes up the whole screen, and its color is changed when the theme is changed

Reply:
Can you clarify which div you're talking about? It would still be great to change the electron background color as it would allow to not have a grey color shown when resizing the window (which is shown the time for the rest of the content to be resized).

Reply:
The div that React renders to.

Reply:
@4ian should I put the code to change the background color in the `index.js` or is there a better file for that?

Reply:
@4ian I was just messing around with resizing the window, and it appears to not be a problem when resizing.

Reply:
@4ian I think I've just about given up on this. You can still add it if you want.